lit_parrot/speed_monitor.py

```diff
@@ -187,11 +187,14 @@ def on_train_batch_end(
         samples: int,  # total samples seen (per device)
         train_elapsed: float,  # total training time (seconds)
         world_size: int,
-        estimated_flops_per_batch: Optional[float] = None,  # (per device)
-        measured_flops_per_batch: Optional[float] = None,  # (per device)
```

Comment:
Removed support for passing both to simplify. The `estimated_flops()` formula is left for reference in the scripts, but not used for the throughput calculation here.